ui/cypress/integration/slow_query/list.compat_spec.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2022 PingCAP, Inc. Licensed under Apache-2.0.
+import { skipOn } from '@cypress/skip-test'
+
+describe('SlowQuery list compatibility test', () => {
+  before(() => {
+    cy.fixture('uri.json').then(function (uri) {
+      this.uri = uri
+    })
+  })
+
+  beforeEach(function () {
+    cy.login('root')
+    cy.visit(this.uri.slow_query)
+    cy.url().should('include', this.uri.slow_query)
+  })
+
+  describe('Available fields', () => {
+    skipOn(Cypress.env('FEATURE_VERSION') !== '6.0.0', () => {
+      it('Show all available fields', () => {
+        cy.intercept('/dashboard/api/slow_query/available_fields').as(
+          'getAvailableFields'
+        )
+        cy.wait('@getAvailableFields')
```

Comment:
I think we don't need to intercept the request manually since when the request will be sent when the slow query page loaded.